eng/Versions.props

```diff
@@ -2,8 +2,8 @@
 <!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
 <Project>
   <PropertyGroup>
-    <VersionPrefix>17.11.0</VersionPrefix>
-    <PackageValidationBaselineVersion>17.8.3</PackageValidationBaselineVersion>
+    <VersionPrefix>17.12.0</VersionPrefix>
+    <PackageValidationBaselineVersion>17.11.0-preview-24318-05</PackageValidationBaselineVersion>
```

Comment:
Should it be published?

Reply:
We discussed which version to put here with @rainersigwald It is published internally, unfortunately, we cannot put here any version published publicly - the latest are 17.10 versions only and we need 17.11 here.

Reply:
There's a follow-up item in the release checklist to bump this to the final released version when it's published to nuget.org.